metals/src/main/scala/scala/meta/internal/metals/ForwardingMetalsBuildClient.scala

```diff
@@ -40,6 +41,7 @@ final class ForwardingMetalsBuildClient(
     didCompile: CompileReport => Unit,
     onBuildTargetDidCompile: BuildTargetIdentifier => Unit,
     onBuildTargetDidChangeFunc: b.DidChangeBuildTarget => Unit,
+    onError: String => Future[Unit],
```

Comment:
```suggestion
    handler: BSPErrorHandler,
```
maybe? This will be easier to navigate and a clear dependency